distributed/tests/test_core.py

```diff
@@ -150,29 +152,58 @@ def listen_on(cls, *args):
             yield assert_can_connect(server.address)
             yield assert_can_connect_locally_6(server.port)
 
+    # InProc
 
-def test_rpc(loop):
-    @gen.coroutine
-    def f():
-        server = Server({'ping': pingpong})
-        server.listen(8883)
+    with listen_on(Server, 'inproc://') as server:
```

Comment:
Ah, I see.  I was misreading things because of how the diff was laid out.  I thought that you were replacing existing tests with inproc tests.  Looking at the flat file makes it more clear that this isn't occurring.